Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -92,7 +92,7 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
     let initialDataUnpacked = Signal.zip(project, baseReward, refTag, context)
 
     let backing = project.map { $0.personalization.backing }.skipNil()
-    self.showPledgeOverTimeUI = project.signal.map { $0.isPledgeOverTimeAllowed }
+    self.showPledgeOverTimeUI = project.signal.map { $0.isPledgeOverTimeAllowed ?? false }
```

Comment:
Shouldn't this be `($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled`?

Reply:
At this point I was using it mostly to check that the mapping of the values ​​was correct, I think I could add it in the next ticket when implementing the UI [MBL-1814](https://kickstarter.atlassian.net/browse/MBL-1814)